server/helpers/paginate.js

```diff
@@ -0,0 +1,33 @@
+import models from '../db/models';
+import utils from './Utilities';
+
+/**
+ * @Module Paginate
+ * @description Controlls all activities related to Articles
+ */
+class Paginate {
+  /**
+   * @description paginate all articles
+   * @param {Object} req - request object
+   * @param {Object} res - response object
+   * @returns {Object} returns articles base on the pagination params
+   * @memberof Paginate
+   */
+  static async paginateArticles(req, res) {
+    let { page, limit } = req.query;
+    limit = parseInt(limit, 10) ? limit : 5;
+    page = parseInt(page, 10) > 0 ? page : 1;
+    const offset = (page - 1) * limit;
+    const articles = await models.Article.findAndCountAll({
+      where: {},
```

Comment:
I'm curious bro. Is there any reason why you specified an empty where constraint?

Reply:
Thank you very much for that question. The reason is that am not supposed to filter the returned article by any condition. If the story, for instance, required that I paginate the article and return articles that belong to a particular author, then I can include a condition in the `where` object. (e.g `where: {authorId: req.user.id)`, then this will return only articles created by an author with the specified authorId.